subprojects/core/src/main/java/org/gradle/internal/cleanup/BuildOutputCleanupRegistry.java

```diff
@@ -26,7 +26,7 @@
     void registerOutputs(Object files);
 
     /**
-     * Returns all registered outputs.
+     * Determines if a file can be cleaned up.
```

Comment:
If we have Javadoc here, we cold as well explain the rules of safely deleted outputs here.